integr8ly/gitea-operator
# Remove deploy proxy option

## Description 
Remove the `deployProxy` option from the custom resource and any related information and installation of OAuth Proxy in the templates, controllers and README files as we no longer require the deployment of OAuthProxy.

Updated the version of the gitea-operator image to `0.0.2`.

## Verification Steps
1. Login to OpenShift
2. Build the operator image to your own repo. Run `make build-and-push ORG=<yourOrg>`
3. Run `make install`
4. Run `make deploy`
5. Run `make create-examples`
6. Ensure that the Gitea operator is successfully deployed.
7. Ensure that there is no OAuth Proxy available within the namespace.
8. Ensure that you can register/login to Gitea.

## Additional Notes
Related PR: https://github.com/integr8ly/gitea-operator/pull/10
